src/lib/y2partitioner/widgets/disk_table.rb

```diff
@@ -0,0 +1,95 @@
+require "yast"
+
+require "cwm/table"
+
+require "y2partitioner/icons"
+require "y2partitioner/device_graphs"
+require "y2partitioner/widgets/blk_devices_table"
+
+module Y2Partitioner
+  module Widgets
+    # Table widget to represent given list of Y2Storage::Disk|Y2Storage::Partition.
+    class DiskTable < CWM::Table
+      include BlkDevicesTable
+
+      # @param disks [Array<Y2Storage::Disk|Y2Storage::Partition>] devices to display
+      # @param pager [CWM::Pager] table have feature, that double click change content of pager
```

Comment:
"Table have feature" is ****ing broken English, BUT I can teach you the importance of articles here:
- "A table" is any table, any table in Yast or in this module
- "The table" is a definite table, like this class

and in this yard  comment it does matter which one it is.